packages/js-client/src/types.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  ContextParams,
+  ContextState,
+  // OverridenState,
+  VersionTag,
+} from "@aragon/sdk-client-common";
+
+// TODO
+// import from @aragon/sdk-client-common
+type OverridenState = {
```

Comment:
Why don't we pull this from client common?

Reply:
Its not exported, thats why the todo is there, I wanted to finish the template and gather the missing pieces from the sdk-common-client to add them all in one commit

Reply:
I would personally favor getting things cohesive here before relying on other tasks to improve the base layout, when the mental context has shifted to something different